### PR TITLE
Add angular-storage@0.0.15 w/ npm auto-update

### LINKED ISSUE #9380: [Request] Add angular-storage
**Library name:** angular-storage
**Git repository url:** https://github.com/auth0/angular-storage
**npm package url(optional):** https://www.npmjs.com/package/angular-storage
**License(s):** MIT
**Official homepage:** https://github.com/auth0/angular-storage
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by NiniHsu**
close #9380, cc @mgonto

Pull request for issue: #9380
Related issue(s): # #
@kennynaoh please help me review this PR, thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/auth0/angular-storage
 * Official website (optional, not the repository): https://github.com/auth0/angular-storage
 * NPM package url (optional): https://www.npmjs.com/package/angular-storage
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 103
   - Count of stars: 663
   - Count of forks: 81
 * NPM download stats (optional):
   - Downloads in the last day: 448
   - Downloads in the last week: 2096
   - Downloads in the last month: 9043

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by NiniHsu** [comment]
@kennynaoh please review the PR, thank you!

File: ajax/libs/angular-storage/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "angular-storage",
+  "filename": "angular-storage.min.js",
+  "version": "0.0.15",
+  "description": "A Storage done right for AngularJS",
+  "keywords": [
+    "angular-storage"
```
Comment (pvnr0082t) [inline ajax/libs/angular-storage/package.json]: @NiniHsu Please give more keywords, three or more will be better.

**Comment by pvnr0082t** [comment]
Note that v0.0.7 can not be grabbed from NPM.

**Comment by NiniHsu** [comment]
@pvnr0082t I add some keywords. Please review the PR, thanks!

**Pull request merged**